src/osdag/design_type/compression_member/Column.py

```diff
@@ -912,33 +934,22 @@ def design_column(self):
                 self.list_yy.append(section)
 
                 # Step 1 - computing the effective sectional area
-                self.section_class = self.input_section_classification[section]
+                self.section_class = self.input_section_classification[section][0]
 
                 if self.section_class == 'Slender':
-                    logger.warning("The trial section ({}) is Slender. Computing the Effective Sectional Area as per Sec. 9.7.2, "
-                                   "Fig. 2 (B & C) of The National Building Code of India (NBC), 2016.".format(section))
+                    #logger.warning("The trial section ({}) is Slender. Computing the Effective Sectional Area as per Sec. 9.7.2, "
+                    #            "Fig. 2 (B & C) of The National Building Code of India (NBC), 2016.".format(section))
```

Comment:
This seems like it should be kept. @parthckaria please comment.

Reply:
It has been added later in the code, the logger should ideally only contain the warnings or errors for the optimum section when all section sizes are selected, and not for all of them causing it to spread out too long.